### PR TITLE
task-F9mdzg4P

### PR DESCRIPTION
**Pull request by BYavorskyj**
Added button and click
Button red with black border.
Added click event for it.
Show alert with text "You are awesome"
@AntonShalenui @vkryviak-ua 
https://trello.com/c/F9mdzg4P/32-menu-add-button-and-click

image

### CODE REVIEW

File: src/client/components/Main.js [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
+  }
+
+  render() {
+    const { username } = this.state;
+    return (
+      <main className="global-main">
+        {username ? <h1>{`Hello ${username}`}</h1> : <h1>Loading.. please wait!</h1>}
+        <img className="main-img" src={ReactImage} alt="react"/>
 
-    componentDidMount() {
```
[the remark this answers is not in the document]
Comment (BYavorskyj) [reply src/client/components/Main.js]: @vkryviak-ua  returned

File: src/client/components/main.css [hunk 2]
```diff
@@ -4,3 +4,8 @@
 .main-img {
     width:200px;
 }
+
+button {
```
Comment (vkryviak-ua) [inline src/client/components/main.css]: Please add namespace

File: src/client/components/Main.js [hunk 3]
```diff
@@ -1,8 +1,12 @@
 import React, { Component } from 'react';
 import ReactImage from '../react.png';
 import './main.css';
+
 export default class Main extends Component {
     state = { username: null };
+    clickOnMe = () => {
+    alert('You are awesome');
```
Comment (vkryviak-ua) [inline src/client/components/Main.js]: Add tab

File: src/client/components/Main.js [hunk 4]
```diff
@@ -13,8 +17,11 @@ export default class Main extends Component {
         const { username } = this.state;
         return (
           <main className="global-main">
-                {username ? <h1>{`Hello ${username}`}</h1> : <h1>Loading.. please wait!</h1>}
-                <img className="main-img" src={ReactImage} alt="react" />
```
Comment (vkryviak-ua) [inline src/client/components/Main.js]: Make same witespace

Comment (vkryviak-ua) [reply src/client/components/Main.js]: Sorry. It is ok

File: src/client/components/Main.js [hunk 5]
```diff
@@ -1,8 +1,12 @@
 import React, { Component } from 'react';
 import ReactImage from '../react.png';
 import './main.css';
+
 export default class Main extends Component {
     state = { username: null };
+    clickOnMe = () => {
+    alert('You are awesome');
+  };
```
Comment (vkryviak-ua) [inline src/client/components/Main.js]: And here also

**Comment by vkryviak-ua** [comment]
Totally wrong place. Do this one more time in Menu component not in Main. Remove this branch

**Pull request closed without merging**